macros/query_tags.sql

```diff
@@ -3,6 +3,27 @@
 {%- endmacro %}
 
 {% macro default__set_query_tag() -%}
+    {# Get session level query tag #}
+    {% set session_query_tag = get_current_query_tag() %}
+    {% set session_query_tag_json = {} %}
```

Comment:
```suggestion
    {% set session_query_tag_json = {} %}
```

let's rename this to 'parsed' because it's not `json` once it's been `fromjson`'d?